hugo/data/teams.yml

```diff
@@ -181,3 +185,8 @@ sections:
         name: Francesc Campoy
         position: VP of Developer Relations
         github: https://github.com/campoy
+        - picture: santiago.png
```

Comment:
I think the identation level should be reduced

Reply:
Sorry, done!